src/Widgets/DataIconWidget/DataIconWidgetComponent.tsx

```diff
@@ -1,23 +1,48 @@
 import {
   ContentTag,
   WidgetTag,
+  connect,
   currentLanguage,
   provideComponent,
   useData,
 } from 'scrivito'
 import { DataIconWidget } from './DataIconWidgetClass'
 import { alignmentClassName } from '../../utils/alignmentClassName'
-import { isDataIconConditionWidget } from '../DataIconConditionWidget/DataIconConditionWidgetClass'
+import {
+  DataIconConditionWidgetInstance,
+  isDataIconConditionWidget,
+} from '../DataIconConditionWidget/DataIconConditionWidgetClass'
 import { IconComponent } from '../../Components/Icon'
 import { ensureString } from '../../utils/ensureString'
 import { useEnumOptions } from '../../utils/useEnumOptions'
 
 provideComponent(DataIconWidget, ({ widget }) => {
-  const attributeValue = ensureString(useData().dataItemAttribute()?.get())
-
-  const size = widget.get('size') || 'bi-2x'
+  return (
+    <WidgetTag className={alignmentClassName(widget.get('alignment'))}>
+      <ContentTag
+        content={widget}
+        attribute="label"
+        className="text-bold opacity-60 text-extra-small text-uppercase"
+      />
+      <AttributeValue
+        conditions={widget.get('conditions').filter(isDataIconConditionWidget)}
+        fallbackIcon={widget.get('fallbackIcon') || 'bi-question-octagon'}
+        size={widget.get('size') || 'bi-2x'}
+      />
+    </WidgetTag>
+  )
+})
 
-  const conditions = widget.get('conditions').filter(isDataIconConditionWidget)
+const AttributeValue = connect(function AttributeValue({
```

Comment:
* [ ] FYI the name `AttributeValue` reads a bit strange, as we actually render an icon for the attribute value. How about `Icon`?